Adding the missing helping verb (#23180)
grafana/grafana: docs/sources/features/dashboard/dashboards.md

1. Zoom out time range
2. Time picker dropdown. Here you can access relative time range options, auto refresh options and set custom absolute time ranges.
3. Manual refresh button. Will cause all panels to refresh (fetch new data).

> 4. Dashboard panel. You can edit panels by clicking the panel title.

Reviewer: ```suggestion
4. Dashboard panel. Click the panel title to edit panels.
```

Reviewer: This is an example of more active and direct instructions.
